pkg/socker/socker.go

```diff
@@ -15,8 +20,33 @@ func New() *Socker {
 }
 
 // ListImages would list all available images from your images hub.
-func (s *Socker) ListImages() {
-	fmt.Println("all images")
+func (s *Socker) ListImages(config string) {
```

Comment:
return error, cli program should check error and exit with non-zere code if something wrong happened.